packages/utils/src/index.ts

```diff
@@ -2,7 +2,6 @@ import * as ASTUtils from './ast-utils';
 import * as ESLintUtils from './eslint-utils';
 import * as JSONSchema from './json-schema';
 import * as TSESLint from './ts-eslint';
-import * as TSESLintScope from './ts-eslint-scope';
 
```

Comment:
```suggestion

export * as TSESLintScope from '@typescript-eslint/scope-manager';
```

Do we think this is a good idea? Or should we just tell people to reference the package directly?